topic-operator/src/main/java/io/strimzi/operator/topic/TopicCommandSerde.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.topic;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import io.apicurio.registry.utils.kafka.SelfSerde;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+/**
+ * TopicCommand Kafka Serde
+ */
+public class TopicCommandSerde extends SelfSerde<TopicCommand> {
+
+    private static final String UUID = "uuid";
+    private static final String TYPE = "type";
+    private static final String TOPIC = "topic";
+    private static final String KEY = "key";
+
+    @Override
+    public byte[] serialize(String topic, TopicCommand data) {
+        return TopicSerialization.toBytes((mapper, root) -> {
+            root.put(UUID, data.getUuid());
```

Comment:
Do we have version elsewhere?

Reply:
So you would be able to properly (de)serialize things based on the version?
Is this really necessary? Or we could just adjust the code to understand any potential changes in the future.

Reply:
So that deserialization is unambiguous, indeed. Kafka does this religiously everywhere data gets persisted. I know we don't expect this to change and the argument can be made that no version => version 0, but I'd still prefer it.